<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackJackTable</class>
 <widget class="QMainWindow" name="BlackJackTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackJackTable</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="tablePic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1152</width>
      <height>648</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../OneDrive/Pictures/BlackJackTable.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayGameButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Play Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Hit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stayButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stay</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>365</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="playerCounter">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="playerLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>How many Players?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>User Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stats Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playerConfirmationButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Let's Play!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="statsProgress">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>What is your name?</string>
    </property>
   </widget>
   <widget class="QLabel" name="runningLabel">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>340</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Running...</string>
    </property>
   </widget>
   <widget class="QLabel" name="userHandScoreLabel">
    <property name="geometry">
     <rect>
      <x>934</x>
      <y>460</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current hand:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doubleDownButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Double Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeckLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>How many decks?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="deckCountSpinBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="minBetLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Bet: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minBetLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>320</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="buyInLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy-In:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="buyInLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NextRoundButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Round</string>
    </property>
   </widget>
   <widget class="QLabel" name="BetLabel">
    <property name="geometry">
     <rect>
      <x>474</x>
      <y>265</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Place your bet:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BetLineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="moneyLabel">
    <property name="geometry">
     <rect>
      <x>934</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Money: </string>
    </property>
   </widget>
   <widget class="QLabel" name="card1Label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>330</y>
      <width>82</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card2Label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>82</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitGameButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="card3Label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>82</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="card4Label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>210</y>
      <width>82</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
